keccak-air/examples/prove_baby_bear_sha256.rs

```diff
@@ -65,10 +66,8 @@ fn main() -> Result<(), impl Debug> {
     type Pcs = TwoAdicFriPcs<Val, Dft, ValMmcs, ChallengeMmcs>;
     let pcs = Pcs::new(dft, val_mmcs, fri_config);
 
-    type MyConfig = StarkConfig<Pcs, Challenge, ByteHash, Challenger>;
-    let config = MyConfig::new(pcs, byte_hash, |byte_hash| {
-        Challenger::from_hasher(vec![], byte_hash)
-    });
+    type MyConfig = StarkConfig<Pcs, Challenge, Challenger>;
+    let config = MyConfig::new(pcs, challenger);
```

Comment:
This can be made a bit more succinct, though arguably the current version is clearer. Up to you whether to change it or not.
```suggestion
    let config = StarkConfig::<_, Challenge, _>::new(pcs, challenger);
```
Same goes in all the other examples.

Reply:
Yeah given these are all examples, I think sticking with clarity over succinctness makes sense for now.